packages/extension/src/entrypoints/options/index.html

```
@@ -4,6 +4,8 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Configuration - Empower Poster</title>
+  <-- For wxt manifest.json auto-generation: -->
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </--> ], start tag match failed [ <-- For wxt manifest.json auto-generation: --> ] on line 7.

(tag-pair)